todo_list/views.py

```diff
@@ -0,0 +1,68 @@
+from django.http import (
+    HttpRequest,
+    HttpResponseRedirect,
+    HttpResponse
+)
+from django.shortcuts import get_object_or_404
+from django.urls import reverse_lazy
+from django.views import generic
+from todo_list.forms import TaskForm
+from todo_list.models import Task, Tag
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports) 